SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -488,32 +491,41 @@ bool IsSupportedClass(const UClass* SupportedClass)
 }
 
 
-TSet<UClass*> GetAllSupportedClasses()
+TSet<UClass*> GetAllSupportedClasses(const TArray<UObject*>& AllClasses)
 {
 	TSet<UClass*> Classes;
-	
-	for (TObjectIterator<UClass> ClassIt; ClassIt; ++ClassIt)
+
+	for(const auto& ClassIt : AllClasses)
 	{
-		UClass* SupportedClass = *ClassIt;
+		UClass* SupportedClass = Cast<UClass>(ClassIt);
 
+		if (SupportedClass != nullptr)
```

Comment:
Is this required since `IsSupportedClass` checks for `.IsValid()` and then prints an appropriate error message?

Reply:
Makes sense, will remove this check

Reply:
Removed.